### PR TITLE
Add warning comment placeholder (7.9 branch)

### PR DESCRIPTION
**Pull request by nkammah**
The development warning comment will help with the automation. 

I will backport the PR against `6.8`, _main branch_, `7.x` once this one is reviewed and merged.

### CODE REVIEW

**Review by jmlrt — APPROVED** [review]
LGTM⛴

**Comment by jmlrt** [comment]
> I will backport the PR against `6.8`, _main branch_, `7.x` once this one is reviewed and merged

FYI, the expected workflow is to create the PR against _main branch_, then to backport it on `7.x`, `7.9`, and `6.8`

**Review by jmlrt — CHANGES_REQUESTED** [review]
Markdown preview is uncorrect:

Screenshot 2020-11-04 at 10 12 04

File: apm-server/README.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ The design and code is less mature than official GA features and is being
 provided as-is with no warranties. Alpha features are not subject to the support
 SLA of official GA features (see [supported configurations][] for more details).
 
-**Warning**: This branch is used for development, please use the latest [7.x][] release for released version.
+<!-- development warning -->**Warning**: This branch is used for development, please use the latest [7.x][] release for released version.
```
Comment (jmlrt) [inline apm-server/README.md:11]:
```
<!-- development warning placeholder -->
**Warning**: This branch is used for development, please use the latest [7.x][] release for released version.
```

I have 2 suggestions:
- maybe add "placeholder" in the markdown comment to make it more explicit what it is, especially when the warning is removed in released version
- maybe move placeholder comment and warning statements on different lines to fix the markdown format issue

**Review by jmlrt — APPROVED** [review]
LGTM⛴

**Comment by nkammah** [comment]
jenkins test this please

**Pull request merged**